coldfront/core/project/templates/project/project_detail.html

```diff
@@ -151,6 +151,45 @@ <h3 class="d-inline" id="users"><i class="fas fa-users" aria-hidden="true"></i>
 </div>
 <!-- End Project Users -->
 
+<!-- Start Project Attributes -->
+<div class="card mb-3">
+  <div class="card-header">
+    <h3 class="d-inline"><i class="fas fa-info-circle" aria-hidden="true"></i> Attributes</h3> <span class="badge badge-secondary">{{attributes.count}}</span>
+    <div class="float-right">
+      {% if project.status.name != 'Archived' and is_allowed_to_update_project %}
```

Comment:
Managers do not have permissions to create project attributes. Only admin are able to at the moment.